package.json

```diff
@@ -25,5 +25,8 @@
     "eslint-plugin-standard": "^4.0.1",
     "stylelint": "^13.7.1",
```

Comment:
stylelint can be removed. also the config package and .stylelintrc if it's still there.